src/features/pages/deriv-ez/deprecated-other-platforms/index.tsx

```diff
@@ -1,9 +1,5 @@
 import React, { useEffect, useState } from 'react'
-import dxIcon from 'images/svg/deriv-ez/dx-icon.svg'
-import dtIcon from 'images/svg/deriv-ez/dt-icon.svg'
-import dmt5Icon from 'images/svg/deriv-ez/dmt5-icon.svg'
-import dGoIcon from 'images/svg/deriv-ez/go-icon.svg'
-import dbIcon from 'images/svg/deriv-ez/db-icon.svg'
+import { DbIcon, DGoIcon, Dmt5Icon, DtIcon, DxIcon } from 'images/svg/deriv-ez'
```

Comment:
Can you cross this with @meenakshi-deriv's PR I think we are having thesame icon. The purpose of this initiative is also to identify icon or image duplications.